### DISCUSSION TITLE
My evaluation period is over, i changed account and want to use it again. Is it possible?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#50791] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I had Copilot extension in VS Code and it worked fine during evaluation period. Then I got another account (the one I using right now ArtemRevaEpam) and expected to apply for free evaluation once again.
But copilot extension does not work anymore because: "evaluation is expired".

Here is logs:
[INFO] [auth] [2023-03-23T08:49:04.027Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-03-23T08:49:04.032Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."

I have "GitHub Pull Requests and Issues" extension with my current ArtemRevaEpam account and it works fine.

I tried:
- Tried to uninstall Copilot ext., reload VS Code, install Copilot again. 
- Enabled everything I could in github settings (see screenshot attached). 
- Checked if extension enabled in VS Code. Checked ext, version (latest)
- Checked for VS Code version (latest)

Worth to mention 
- For some reason i don't have Copilot icon on activity bar (don't know why)
- I did not offered to login in Copilot
- I don't have any copilot env variables or github env variables (not sure if they should exist)
- I work from home (Turkey)
- 2FA is enabled for account
- Windows 10, VS Code 1.76.2, Copilot ext. v1.78.9758, 

Is it possible to use another evaluation period with another account? (i am not cheating, i need it for my job).
If yes, what can i do to run it locally?

![image](https://user-images.githubusercontent.com/123643768/227153534-6ea85544-259e-4318-8e3f-0f45ae226fea.png)

**Comment by Akash1134** [comment, marked as answer]
Hi @ArtemRevaEpam, We're sorry for the trouble here. The best place to go for answers to account-specific questions like this one is by opening a ticket on our Support page here: https://support.github.com/contact Please open a ticket with them and they should be able to help you get this sorted :)

**Reply by ghost** [reply]
thank you, will do it
